src/centos/7/mlnet/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+
+RUN yum install -y \
+        perl-Data-Dumper
+
+RUN wget http://releases.llvm.org/3.9.1/openmp-3.9.1.src.tar.xz && \
```

Comment:
A comment here would be valuable IMO just calling out the purpose of the layer - install OpenMP.  This was called out in the Ubuntu Dockerfile and that was more obvious why not here as well😸.

Reply:
Thanks Michael - since the opensuse issue causes the build to fail, its blocking this PR from going in.

Reply:
...or maybe I dont have permissions to merge? I will update the PR per your comment.

Reply:
Not everyone has permission to merge.  I will merge once all legs are green except for opensuse.